.github/workflows/build.yml

```diff
@@ -0,0 +1,60 @@
+name: Physics Server Box2D Builds
+
+on:
+  push: {}
+  pull_request: {}
+
+jobs:
+  Build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [template_debug]
+        os: [macos-latest, ubuntu-latest, windows-latest]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+      - name: Load .scons_cache directory
+        uses: actions/cache/restore@v3
+        with:
+          path: |
+            ${{ github.workspace }}/.scons-cache/
+            ${{ github.workspace }}/godot-cpp/.scons-cache/
+          key: ${{ matrix.os }}-${{ matrix.target }}
+      - name: Setup python and scons
+        uses: ./.github/actions/deps
+      - name: Lint
+        shell: sh
+        run:
+          clang-format
```

Comment:
It would be nice to run this with `-i` and check the `git diff` as in Godot's [`clang-format.sh`](https://github.com/godotengine/godot/blob/950f4c3b41acef0403618dd6169be90530dea5da/misc/scripts/clang_format.sh).

I would prefer to make the empty diff a requirement, but we can do that later if you prefer.

Reply:
Fixed